azimuth/types/model_performance.py

```diff
@@ -87,7 +87,7 @@ class MetricsPerFilterAPIResponse(MetricsPerFilterModuleResponse):
 
 class ConfusionMatrixResponse(ModuleResponse):
     confusion_matrix: Array[float] = Field(..., title="Confusion Matrix normalized")
-
+    normalized: bool = Field(..., title="normalized state of Confusion Matrix")
```

Comment:
Black (in the tests) fails because you removed that empty line.
```suggestion
    normalized: bool = Field(..., title="Normalized state of Confusion Matrix")

```

Reply:
yes important! changes can be made

Reply:
The pre-commits should catch that.